go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/cilium/cilium
 
 // renovate: datasource=golang-version depName=go
-go 1.21.3
+go 1.21.0
```

Comment:
Should this be `1.21` instead of `1.21.0`?

Reply:
No, according to upstream, `go 1.21` means development versions of Go ([context](https://github.com/golang/go/issues/62278#issuecomment-1693538776)) whereas `go 1.21.0` means to require 1.21.0 or newer stable releases ([context](https://github.com/golang/go/issues/63522#issuecomment-1760201533)).